### PR TITLE
feat: send width height length during export

### PR DESCRIPTION
**Pull request by joerivanveen**
INT-1775

### CODE REVIEW

File: src/Shipment/Request/PostShipmentsRequest.php [hunk 1]
```diff
@@ -219,6 +219,27 @@ private function getPickupLocation(Shipment $shipment): ?array
...
+    private function encodePhysicalProperties(Shipment $shipment): array
+    {
+        $physicalProperties = $shipment->physicalProperties;
+
+        return Utils::filterNull([
+            'weight' => $this->getWeight($shipment),
+            'length' => $physicalProperties ? $physicalProperties->length : null,
+            'width'  => $physicalProperties ? $physicalProperties->width : null,
+            'height' => $physicalProperties ? $physicalProperties->height : null,
+        ]);
```
[the remark this answers is not in the document]
Comment (FreekVR) [reply src/Shipment/Request/PostShipmentsRequest.php:240]: Please add the test as suggested. You added a test to assert storage works, but nothing asserts it gets correctly sent to the API, even though there are some edge cases that should be covered (empty string, "0", "-1" behavior)

**Review by FreekVR — CHANGES_REQUESTED** [review]

File: src/Shipment/Request/PostShipmentsRequest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (joerivanveen) [reply src/Shipment/Request/PostShipmentsRequest.php:240]: Added test.

**Review by FreekVR — APPROVED** [review]

File: src/Shipment/Request/PostShipmentsRequest.php [hunk 2]
```diff
@@ -219,6 +219,27 @@ private function getPickupLocation(Shipment $shipment): ?array
         ]);
     }
 
+    /**
+     * Dimensions are in centimeters, the same unit the API expects and the admin shows, so they are
```
Comment (GravendeelJochem) [inline src/Shipment/Request/PostShipmentsRequest.php:223]: Hoezo doen we dit in CM? Was het niet altijd dat de API alles in MM verwacht? Volgens mij is dit in de SDK wel zo

**Review by GravendeelJochem — CHANGES_REQUESTED** [review]

File: src/Shipment/Request/PostShipmentsRequest.php (hunk 2 above)
Comment (NabDevs) [reply src/Shipment/Request/PostShipmentsRequest.php:223]: @GravendeelJochem The API expects CM. The SDK doesn't do conversion (only does a type-check)

**Review by GravendeelJochem — APPROVED** [review]
